docs/source/data-access-api/index.rst

```diff
@@ -64,6 +78,8 @@ The API can be accessed from the following URL: https://api.dev.imap-mission.com
    :group:
    :include: /query
 
+*Note:* ``filename``, ``N days ago`` *, and* ``today`` *are all [WIP].*
```

Comment:
I think for clarity sake it may be better to note these as WIP in their _descriptions_ (or tag it onto the  _name_ ) in `openapi.yml`.